movo_robot/movo_bringup/launch/movo2.launch

```diff
@@ -21,7 +21,7 @@
 	<!-- Time synchonization for sibot2 -->
 	<node pkg="si_utils" type="sync_chrony" 
 		args="10.66.171.2"
-		name="time_sync" output="screen"/>
+		name="time_sync" launch-prefix="sudo" output="screen"/>
```

Comment:
nice!. so this asks us to enter the passwd when we launch ros2.launch right? and only uses root priviliged for the sync chrony node?

Reply:
By editing the visudo file with "sudo visudo", we are able to allow sudo to run the specific script without a password. This solution is elegant and also safe. See this PR for more details https://github.com/nightingale-project/kinova-movo/pull/6#issuecomment-1279790947